src/components/permissions/loginHelpers.ts

```diff
@@ -105,6 +106,9 @@ const getUserInfo = async (userOrgParameters: UserOrgParameters) => {
       isManager,
     },
     orgList,
+    tokenExpiry:
+      parseInt(String(Date.now() / 1000)) +
+      (config.logoutAfterInactivity ?? DEFAULT_LOGOUT_TIME) * 60,
```

Comment:
We're returning the expiry time to the front-end, although I'm not actually using it (the front-end knows how long the token lifetime is via preferences and refreshed accordingly)